src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,272 @@
+              title: "eIDAS",
+              imageSrc: "/eidas.png"
+          },
+          url: "#",
+        },
+        {
+          identifier: "anonymous",
+          label: "Begin form",
+        }
+      ],
+      loginAsGemachtigdeOptions: [{
+	      identifier: "digid_machtigen",
+	      label: "DigiD Machtigen",
+        logo: {
+            href: "https://www.digid.nl/",
+            title: "DigiD Machtigen",
+            imageSrc: "/digid.png"
+        },
+        url: "#",
+      }]
+    }}
+  >
+    {DisplayTemplate.bind({})}
+  </Story>
+</Canvas>
+
+Note that the anonymous form start option is implemented as a button with `type="submit"` and is therefore
+expected to be nested inside of a `<form>` element with an appropriate `onSubmit` handler.
+
+### Props
+
+<ArgsTable of={LoginOptionsDisplay} />
+
+## Functional
+
+The `LoginOptions` component handles the 'smart' part of the component. It separates the login options between
+the ones that can be used to 'login as yourself' and the ones that can be used to log in 'on behalf of someone else'.
+It also formats the labels that will be displayed on the buttons and creates the right URL to which the user will be
+redirected upon clicking one of the buttons.
+
+export const LiteralDecorator = (Story, { args }) => (
+  <LiteralsProvider literals={{'beginText': {resolved: args.beginText}}}>
+    <Story />
+  </LiteralsProvider>
+);
+
+export const Template = ({loginRequired=false, loginOptions, onFormStart}) => (
+  <LoginOptions
+    form={{
+      uuid: '81a22589-abce-4147-a2a3-62e9a56685aa',
+      name: 'Test form',
+      loginRequired,
+      loginOptions,
+      translationEnabled: false,
+      autoLoginAuthenticationBackend: '',
+      paymentRequired: false,
+      paymentOptions: [],
+      literals: {
+        previousText: {resolved: 'Previous page', value: ''},
+        beginText: {resolved: 'Begin form', value: ''},
+        changeText: {resolved: 'Change', value: ''},
+        confirmText: {resolved: 'Confirm', value: ''},
+      },
+      slug: 'test-form',
+      url: `<placeholder>`,
+      steps: [],
+      showProgressIndicator: true,
+      maintenanceMode: false,
+      active: true,
+      explanationTemplate: '<p>Toelichtingssjabloon...</p>',
+      submissionAllowed: 'yes',
+      requiredFieldsWithAsterisk: true,
+    }}
+    onFormStart={onFormStart}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Functional"
+    decorators={[LiteralDecorator]}
+    play={async ({args, canvasElement}) => {
+      const canvas = within(canvasElement);
+      let anonymousStartButton = await canvas.getByRole('button');
+      await userEvent.click(anonymousStartButton);
+      await waitFor(() => expect(args.onFormStart).toHaveBeenCalled());
+    }}
+    argTypes={{
+      form: {table: {disable: true}},
+      onFormStart: {
+        action: true,
+      }
+    }}
+    args={{
+      beginText: 'Begin form (anonymous)',
+      loginRequired: false,
```

Comment:
I've added this arg so that we get a nice toggle true/false and see how the story reacts to that, as it has an effect on the container element + additional button.